src/main/java/io/logz/jmx2graphite/GraphiteClient.java

```diff
@@ -1,24 +1,21 @@
 package io.logz.jmx2graphite;
 
-import static io.logz.jmx2graphite.GraphiteProtocol.TCP;
-import static io.logz.jmx2graphite.GraphiteProtocol.UDP;
-
 import com.codahale.metrics.graphite.Graphite;
 import com.codahale.metrics.graphite.GraphiteSender;
 import com.codahale.metrics.graphite.GraphiteUDP;
 import com.google.common.base.Joiner;
 import com.google.common.collect.Lists;
+import org.slf4j.Logger;
```

Comment:
Many imports were added to this file, though no code was changed

Reply:
Nothing was added, it was just reordered by the automatic import optimizer